### ISSUE TITLE
Calming Air in the Cooling the Fury quest can miss

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#6534]
> Calming Air in the Cooling the Fury quest can miss

.go xyz -1140.540000 -4064.570000 102.678000 0 2.265050

**Comment by zana244** [comment]
Correct, it can miss, think it uses spell hit? using a lower level char makes this easy to reproduce. Also, calming air is not marked as "Quest Item" in its tooltip?
